### PR TITLE
Add navigation 

### PR DESCRIPTION
**Pull request by doboianh**

### CODE REVIEW

File: nav.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import {Navbar} from 'react-bootstrap'
+import {Container} from 'react-bootstrap'
+import {Nav} from 'react-bootstrap'
+import {NavDropdown} from 'react-bootstrap'
+import {Form} from 'react-bootstrap'
+import {FormControl} from 'react-bootstrap'
+import {Button} from 'react-bootstrap'
```
Comment (doboianh) [reply nav.js]: oh i got it

Comment (siddhantprateek) [inline nav.js]: import all at once 
```
import {
   Navbar,
   Container,
   Nav,
   NavDropdowm,
   Form,
   FormControl,
   Button
} from 'react-bootstrap';
```

File: nav.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <Navbar className="nav">
+            <Container fluid className="nav-content">
+                <Navbar.Brand className="brand" href="#">CryptoApp</Navbar.Brand>
+                <Navbar.Toggle aria-controls="navbarScroll" />
+                <Navbar.Collapse id="navbarScroll">
+                <Nav className="nav-list me-auto my-2 my-lg-0" style={{ maxHeight: '100px' }} navbarScroll>
+                    <Nav.Link className="nav-item" href="#action1">Home</Nav.Link>
+                    <Nav.Link className="nav-item" href="#action1">Create</Nav.Link>
+                    <Nav.Link className="nav-item" href="#action1">Statistic</Nav.Link>
+                    <Nav.Link className="nav-item" href="#action1"> Link </Nav.Link>
```
Comment (siddhantprateek) [inline nav.js:17]: remove this line 23

File: nav.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+function Navigation() {    
+    return (
+        <Navbar className="nav">
+            <Container fluid className="nav-content">
+                <Navbar.Brand className="brand" href="#">CryptoApp</Navbar.Brand>
+                <Navbar.Toggle aria-controls="navbarScroll" />
+                <Navbar.Collapse id="navbarScroll">
+                <Nav className="nav-list me-auto my-2 my-lg-0" style={{ maxHeight: '100px' }} navbarScroll>
+                    <Nav.Link className="nav-item" href="#action1">Home</Nav.Link>
+                    <Nav.Link className="nav-item" href="#action1">Create</Nav.Link>
```
Comment (siddhantprateek) [inline nav.js:15]: remove this line 21

**Review by siddhantprateek — CHANGES_REQUESTED** [review]

**Review by siddhantprateek — APPROVED** [review]

**Comment by siddhantprateek** [comment]
@doboianh thank you for your contribution

**Comment by doboianh** [comment]
> @doboianh thank you for your contribution

thank you for your support.

**Pull request merged**